packages/office-ui-fabric-react/src/components/Calendar/Calendar.tsx

```diff
@@ -67,7 +68,7 @@ export class Calendar extends BaseComponent<ICalendarProps, ICalendarState> impl
     }
 
     this.setState({
-      selectedDate: value || new Date()
+      selectedDate: value || (today || new Date())
```

Comment:
This is in case null value was passed in since default param won't be used in such case.